test/integration/tls_test.py

```diff
@@ -6,18 +6,6 @@
 from pyexasol import ExaConnectionFailedError
 
 
-@pytest.fixture
```

Comment:
Why did you remove this?

Reply:
I removed this one, as it's the same as the fixture in the `test/integration/conftest.py`, so it seemed like we could reduce test setup duplication.